packages/cache/src/index.js

```diff
@@ -87,7 +103,12 @@ let createCache = (options: Options): EmotionCache => {
         const attrib = ((node.getAttribute(`data-emotion`): any): string).split(
           ' '
         )
-        if (attrib[0] !== key) {
+        if (
+          attrib[0] !== key ||
+          // this means we will ignore elements which don't have a space in them which
+          // means that the style elements we're looking at are only Emotion 11 server-rendered style elements
+          attrib.length === 1
+        ) {
```

Comment:
Instead of this if condition we could leverage the browser capabilities and query for this:
```js
document.querySelectorAll(`[data-emotion^="${key} "]`)
```

Reply:
Great call-out!